microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,139 +20,173 @@ ms.date: 05/04/2021
+## Browse resource files by folder in the Page tab of the Sources tool
+
+You can use the **Sources** tool to view the webpage's resource files organized by directory, as follows:
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
+
+1. In DevTools, on the main toolbar, select the **Sources** tab.  If that tab isn't visible, click the **More tabs** (![More tabs icon](../media/more-tabs-icon-light-theme.png)) button.
+
+1. In the **Navigator** pane in the upper left, select the **Page** tab.
 
-    Here is a breakdown of the non-obvious items in the previous figure.
+1. By default, resource files are grouped by folder.  If the resources are all listed alphabetically by file name instead, click the **More options** (...) button to the right of the **Page** tab, and then select **Group by folder**:
+
+   ![The webpage's resource files grouped by folder in the Page tab of the Sources tool](../media/resources-sources-page-empty.msft.png)
+
+    The **Page** tab contains various node types, including:
 
     | Page item | Description |
-    |:--- |:--- |
+    |---|---|
     | `top` | The main document [browsing context](https://developer.mozilla.org/docs/Web/HTML/Element/iframe). |
-    | `airhorner.com` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `comlink.global.js` file is probably `https://airhorner.com/scripts/comlink.global.js`. |
-    | `scripts` | A directory. |
+    | `microsoftedge.github.io` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `buttons.js` file is [https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js](https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js). |
+    | `css`, `photos` | Directories |
     | `(index)` | The main HTML document. |
-    | `sw.js` | A service worker runtime context. |
+    | `buttons.js` | A script to handle button presses on the page. |
 
-1. Select a resource to view it in the **Editor**.
+1. Select a resource file to view it in the Editor of the **Sources** tool:
 
-   ![View a file in the Editor.](../media/resources-sources-page-resource.msft.png)
+   ![Viewing a resource file in the Editor of the Sources tool](../media/resources-sources-page-resource.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse resources by filename in the Sources tool
+## Browse resource files sorted by filename in the Page tab of the Sources tool
 
-By default, the **Page** tab groups resources by directory.  To display the resources for each domain as a flat list, instead of grouping them by directory:
+By default, the **Page** tab in the **Sources** tool groups resource files by folder.  To instead display all the resource files for each domain grouped together in a single alphabetized list:
 
```

Comment:
```suggestion

1. In a new window or tab, go to a webpage, such as the [Accessibility-testing features demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/).  (The source files for that webpage are at [MicrosoftEdge / Demos > devtools-a11y-testing](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing).)

```

Reply:
@dan-wesley - h2 "Browse resource files sorted by filename in the Page tab of the Sources tool" - the Comment for the Suggested change was resolved though not applied (or, not stated whether you intended to apply or not apply the Suggested change).